common/model/query/sort/sort.js

```diff
@@ -0,0 +1,34 @@
+import * as A from '@core/arena'
+
+import { SortCriteria } from '../sortCriteria'
+
+export const create = () => []
+
+// ====== READ
+export const { isEmpty } = A
+export const containsVariable = (variable) => (sort) =>
+  Boolean(sort.find((sortCriteria) => SortCriteria.getVariable(sortCriteria) === variable))
```

Comment:
is find already boolean?

Reply:
yes damn it sorry... will change in the next PR